imagetest/testworkflow.go

```diff
@@ -225,6 +225,70 @@ func (t *TestWorkflow) addStartStep(stepname, vmname string) (*daisy.Step, error
 	return startInstancesStep, nil
 }
 
+func (t *TestWorkflow) addCreateNetworkStep(vmname, networkName string) (*daisy.Step, error) {
+	createNetworks := &daisy.CreateNetworks{}
+	pFalse := false
+	network := &daisy.Network{
+		Network: compute.Network{
+			Name: networkName,
+		},
+		AutoCreateSubnetworks: &pFalse,
+	}
+	*createNetworks = append(*createNetworks, network)
+
+	createNetworksStep, err := t.wf.NewStep("create-network-" + vmname)
```

Comment:
creating a network is not specific to a VM; it should be named after the network name, not the VM name